index.html

```diff
@@ -37,7 +37,7 @@ <h1>yobimoriアプリ利用規約</h1>
         1. 本アプリは、yobimoriデバイス（以下「本デバイス」といいます）起動時に発信される位置情報を本アプリで受信することにより、
         当該本デバイスを占有する漁師等が海難事故に巻き込まれた際の発見を補助するアプリです。<br>
         <br>
-        2. 利用者は、利用者の個人情報及び位置情報について「プライバシーポリシー（リンクに帰る部分）」に従って取り扱うことに同意したものとします。<br>
+        2. 利用者は、利用者の個人情報及び位置情報について「プライバシーポリシー（<a href="/yobimori_privacy_policy/">プライバシーポリシー</a>）」に従って取り扱うことに同意したものとします。<br>

```

Comment:
```suggestion
        2. 利用者は、利用者の個人情報及び位置情報について「<a href="/yobimori_privacy_policy/">プライバシーポリシー</a>」に従って取り扱うことに同意したものとします。<br>

```

これでいいんじゃないでしょうか
いまだと
> 「プライバシーポリシー（[プライバシーポリシー](https://nanofreaks.github.io/yobimori_privacy_policy/)）」
みたいになってしまう